omod/src/main/java/org/openmrs/module/pihcore/RequestMonitoringFilter.java

```diff
@@ -3,26 +3,19 @@
  * Version 1.0 (the "License"); you may not use this file except in
  * compliance with the License. You may obtain a copy of the License at
  * http://license.openmrs.org
- *
+ * <p>
```

Comment:
Strange auto-html formatting? or did the license template change?

Reply:
It's not auto, I added this in just because Intellij gives an error, and this is it's suggested fix.